.appveyor.yml

```diff
@@ -0,0 +1,13 @@
+version: '{build}'
+os: Windows Server 2012
+matrix:
+  fast_finish: true
+
+init:
+  - cmd: choco install sbt
+  - cmd: refreshenv
+build_script:
+  - cmd: sbt dotty-bootstrapped/compile
+test_script:
+  - cmd: sbt "dotty-bootstrapped/dotc tests\pos\HelloWorld.scala"
```

Comment:
May I suggest to add
```
  - cmd: sbt sjsJUnitTests/test
```
as an additional test to improve coverage on the Windows CI?

Reply:
It's done now.